CMakeLists.txt

```diff
@@ -112,6 +142,18 @@ install(
     FILE ${PROJECT_NAME}Targets.cmake
 )
 
+#-----------------------------------------------------------------------
+# Optionally install python library
+if (OPENASSETIO_MEDIACREATION_GENERATE_PYTHON)
+    include(ThirdParty)
```

Comment:
I get where you're coming from, but i'm a little dubious to change the name of code like this when it's the same logic across multiple codebases, I want there to be a mapping so we don't just have to remember that they're meant to be the same.

As for unconditionally including it, that seems like a reduction in readability to me, as you lose the context that it's included for the reason of python. If thirdparty expands, it'll be natural to move it out, but until then I think it's in the appropriate spot.

Reply:
Some good points. :thinking: 

Personally I think `Thirdparty.cmake` being a black box (doing the correct thing for the chosen `option`s) is OK. 

I also think/hope our various `Thirdparty.cmake` files are only coincidentally similar because they converge to best/useful practice. I certainly wouldn't want to force them all to conform at the expense of improvements.

But you do make reasonable arguments, so let's leave as is.